### ISSUE TITLE
Mentioning does not work in bio

### ISSUE THREAD

**Issue by DeusFigendi** [diaspora/diaspora#7933]
Hey,
maybe this works as intended, but I just recognized I cannot mention accounts in a profiles bio.
Of coarse usually nobody wants to **mention** someone so the account gets notified etc. but maybe someone likes to link to another users profile.

So was my case: I created a sock puppet / twink account / bot / test account -thingy (so just an account that isn't my main account) and to make this transparent I wrote to the bio

````
This is an account ran by @{deusfigendi@pod.geraspora.de} to generate some testdata to play with.
````

and this results in 

````
This is an account ran by @<a href="mailto:deusfigendi@pod.geraspora.de" target="_blank" rel="noopener noreferrer">deusfigendi@pod.geraspora.de</a> to generate some testdata to play with.
````

That is interesting because the `{ braces }` were removed :-o

Like I said… maybe it's not meant to be able to mention accounts in bio but as a user I would expect I can mention everywhere I can use markdown.

**Comment by goobertron** [comment]
To me it makes some sense that mentioning doesn't work in bios, because these are not communications in the sense that posts, comments and conversations are, and the person mentioned would thus not be able to respond. And would they get a notification when 'mentioned' in a bio? About what?

The only case I can see for mentions in bios is to make it slightly more convenient to do the markdown `[User Name](path/to/account/guid)`. To me it makes sense not to have mentions enabled here, as there would be no action performed than making a link to the account profile; just use the standard link markdown code instead.

**Comment by Flaburgan** [comment]
Agree, what you want to do here is a link, not a mention, so please use the link syntax ;)

**Comment by SuperTux88** [comment]
> The only case I can see for mentions in bios is to make it slightly more convenient to do the markdown `[User Name](path/to/account/guid)`. To me it makes sense not to have mentions enabled here, as there would be no action performed than making a link to the account profile; just use the standard link markdown code instead.

> what you want to do here is a link, not a mention, so please use the link syntax ;)

I thought the same first, but the relative link @goobertron suggest has the same problems as any relative links on diaspora:

* It only works on diaspora, everywhere else it links to a 404.
* If the linked target isn't known, it links to a 404.

And I think that linking to a profile from a bio is a valid use-case and using something that already exists and users already know maybe isn't that bad? We don't need the notifications, but otherwise a mention would work perfectly. And it would also have advantages for the protocol, because that way every text with Markdown would support the same features/syntax.

An alternative would be to do something with `diaspora://`-URLs but that would be something special again (currently we don't support linking to profiles that way, and also don't support of fetching profiles with it).

Since I think that's a valid feature-request and we don't provide a proper solution for this yet, I'm reopening this.
